content/docs/command-reference/plot/index.md

```diff
@@ -0,0 +1,189 @@
+# plot
+
+Contains commands to visualize
+[continuous metrics](/doc/command-reference/plot#continuous-metrics) in
+structured files like JSON, CSV, TSV: [show](/doc/command-reference/plot/show),
+[diff](/doc/command-reference/plot/diff).
+
+## Synopsis
+
+```usage
+usage: dvc plot [-h] [-q | -v] {show,diff} ...
+
+positional arguments:
+  COMMAND
+    show         Plot data from a file
+    diff         Plot changes between commits in the DVC repository,
+                 or between the last commit and the workspace.
```

Comment:
```suggestion
    show         Generate a plot image file from a continuous metrics file
    diff         Plot continuous metrics differences between commits in the DVC repository,
                 or between the last commit and the workspace.
```
> May need rewrapping
Cc @pared also for the core repo PR

Reply:
Moved to iterative/dvc/pull/3577

Reply:
The definition for show is still pending in the core repo (https://github.com/iterative/dvc/pull/3577#discussion_r416844217)
But the one for diff has been applied ✅

Reply:
Oh I see this one has been applied. Thanks, resolving.